.woodpecker/test.yml

```diff
@@ -50,6 +50,15 @@ pipeline:
     image: mstruebing/editorconfig-checker
     group: test
 
+  securitycheck:
+    group: test
+    image: aquasec/trivy:latest
+    commands:
+      - trivy fs --exit-code 0 --skip-dirs web/ --severity UNKNOWN,LOW .
```

Comment:
we sould skip docs too and add a step for it to docs pipeline

Reply:
done